index.html

```diff
@@ -96,7 +96,10 @@ <h5>Research Interests</h5>
                 {{/if}}
                 {{#if has_news_stories}}
                   <li class="tabs-title"><a href="#news" data-tabs-target="news">News</a></li>
-                {{/if}}                
+                {{/if}}
+                {{#if has_other_publications}}
+                  <li class="tabs-title"><a href="#other_publications" data-tabs-target="other-publications">Others</a></li>
```

Comment:
The only other thing that I'm not sure about here is the title of this tab. I can't really suggest anything better than "Other Publications" - or perhaps "Other Works" - but just "Others" doesn't seem descriptive enough given the context.

Reply:
Okay, I'll try one of those out.

Reply:
I agree though that it's not very descriptive